src/chart/options/channels.h

```diff
@@ -15,6 +15,7 @@ namespace Vizzu::Gen
 struct Channels : Refl::EnumArray<ChannelId, Channel>
 {
 	using IndexSet = std::set<Data::SeriesIndex>;
+	EnumArray<AxisId, AxisChannelProperties> axisProps;
```

Comment:
Wouldn't something like this would be more straight forward:
```
struct Channels
{
	Refl::EnumArray<NonAxisId, Channel> nonAxisChannels;
	EnumArray<AxisId, AxisChannel /* inherits Channel */> axisChannels;

Reply:
Yes. Later I'll fix that.

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/632/commits/bbbc5238e656e889b13c997566ff6cb833ae195d